diff --git a/vendor/jayktaylor/mw-discord/src/DiscordHooks.php b/vendor/jayktaylor/mw-discord/src/DiscordHooks.php
index: vendor/jayktaylor/mw-discord/src/DiscordHooks.php
--- a/vendor/jayktaylor/mw-discord/src/DiscordHooks.php
+++ b/vendor/jayktaylor/mw-discord/src/DiscordHooks.php
@@ -4,6 +4,7 @@
 use MediaWiki\MediaWikiServices;
 use MediaWiki\Revision\RevisionRecord;
 use MediaWiki\Storage\EditResult;
+use MediaWiki\Title\Title;
 use MediaWiki\User\UserIdentity;
 
 /**
